be/src/olap/task/engine_publish_version_task.cpp

```diff
@@ -189,6 +191,7 @@ Status EnginePublishVersionTask::finish() {
                     if (tablet->check_version_exist(version)) {
                         continue;
                     }
+                    not_continuous_mow_tablets.insert(tablet_info.tablet_id);
```

Comment:
move it to L196, inside `handle_version_not_continuous`

Reply:
update,  ignore all tablet error when met PUBLISH_VERSION_NOT_CONTINUOUS error